### DISCUSSION TITLE
flet run throws an error in an initial app created with command `flet create .`

### DISCUSSION THREAD

**Discussion by diego-garro** [flet-dev/flet#4424] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

Can't run the initial app because `flet run` throws the following error:

```console
/home/{user}/.flet/bin/flet-0.24.1/flet/flet: error while loading shared libraries: libmpv.so.1: cannot open shared object file: No such file or directory
```

### Code sample

The initial app.

```python
import flet as ft


def main(page: ft.Page):
    page.add(ft.SafeArea(ft.Text("Hello, Flet!")))


ft.app(main)

```

### To reproduce

* create app with poetry: `poetry new my-app`
* add flet dependency: `poetry add flet`
* add the code above to src/main.py
* run the app: `flet run`

### Expected behavior

just run the app...

### Screenshots / Videos

<details open>
<summary>Captures</summary>

[Upload media here]

</details>

### Operating System

Linux

### Operating system details

Fedora 39

### Flet version

0.24.1

### Regression

No, it isn't

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by ndonkoHenri** [comment]
Install the latest Flet pre-release and retry: `pip install flet --pre -U`

You might also want to give https://github.com/flet-dev/flet/issues/2823 a look.
